### PR TITLE
Automatically generate documentation for prometheus-metrics

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- add makefile target to generate documentation `update_metrics_docs`

**Related issue(s)**

### CODE REVIEW

File: components/kyma-metrics-collector/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The basic flow for KMC is as follows:
  * KMC adds the runtimes to a queue to work through them and re-queues a runtime should an error occur. 
  * Information on PVCs, SVCs and Nodes is retrieved from SAP Kyma Runtime (SKR). 
  * This information is sent to EDP as an event stream.
- * For every process step, internal metrics are collected with [Prometheus](https://prometheus.io/docs/introduction/overview/) and alerts have been configured to trigger if any part of the functionality malfunctions.
+ * For every process step, internal metrics are collected with [Prometheus](https://prometheus.io/docs/introduction/overview/) and alerts have been configured to trigger if any part of the functionality malfunctions. See [here](metrics.md)
```
Comment (NHingerl) [inline components/kyma-metrics-collector/README.md:13]:
```suggestion
 * For every process step, internal metrics are collected with [Prometheus](https://prometheus.io/docs/introduction/overview/) and alerts have been configured to trigger if any part of the functionality malfunctions. See the [metrics.md](metrics.md) file.
```

**Review by NHingerl — COMMENTED** [review]
I like meaningful link titles ;)

**Review by NHingerl — APPROVED** [review]

File: components/kyma-metrics-collector/Makefile [hunk 2]
```diff
@@ -34,3 +34,6 @@ go-mod-check-local:
 
 test-alerts:
 	promtool test rules ${PROMETHEUSRULES_PATH}/alerts_test.yaml
+
+update_metrics_docs:
```
Comment (mfaizanse) [inline components/kyma-metrics-collector/Makefile:38]: Where will this make target be triggered? Or is this the same setup as in kyma repository where governance job fails?

Comment (k15r) [reply components/kyma-metrics-collector/Makefile:38]: atm there is no such thing... similar to eventing-controllers metric documentation generation... 
but that is a good point. we should introduce it

**Review by mfaizanse — APPROVED** [review]

**Pull request merged**